cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -168,7 +169,9 @@ private PublicationContext buildPublicationContext()
 
     private Book buildBookForPublicationContext() throws MalformedURLException, InvalidIsbnException {
         List<String> isbnList = new ArrayList<>();
-        isbnList.add(extractIsbn());
+        if (nonNull(extractIsbn())) {
```

Comment:
Suggestion:
```
 private Optional<String> extractIsbn() {
        return Optional.ofNullable(extractCristinBookReport().getIsbn());
    }

    private Book buildBookForPublicationContext() throws MalformedURLException, InvalidIsbnException {
        List<String> isbnList =extractIsbn().stream().collect(Collectors.toList());
```
Optional indicates that a value may or may not be there. I think it reflects more the decision we are taking to accept empty ISBN values. we should start using more this pattern that checking for null here and there.